Source/WebCore/dom/ContainerNode.h

```diff
@@ -85,9 +85,9 @@ class ContainerNode : public Node {
         enum class AffectsElements : uint8_t { Unknown, No, Yes };
 
         ChildChange::Type type;
-        Element* siblingChanged;
-        Element* previousSiblingElement;
-        Element* nextSiblingElement;
+        CheckedPtr<Element> siblingChanged;
```

Comment:
Oh yes, this part definitely regresses Speedometer :(

Reply:
It's probably suppression annotation.